modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/utils/getDefaultEditorConfig.ts

```diff
@@ -0,0 +1,267 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {
+	Alignment,
+	BlockQuote,
+	Bold,
+	EditorConfig,
+	Essentials,
+	Font,
+	GeneralHtmlSupport,
+	Heading,
+	HorizontalLine,
+	Image,
+	ImageBlock,
+	ImageCaption,
+	ImageInline,
+	ImageResize,
+	ImageStyle,
+	ImageToolbar,
+	Indent,
+	Italic,
+	Link,
+	List,
+	MediaEmbed,
+	Paragraph,
+	RemoveFormat,
+	SourceEditing,
+	Strikethrough,
+	Style,
+	Table,
+	TableCaption,
+	TableProperties,
+	TableToolbar,
+	Underline,
+} from 'ckeditor5';
+import {sub} from 'frontend-js-web';
+
+import ItemSelector from '../plugins/ItemSelector';
+import {EEditorConfigPreset, EEditorType} from './types';
+
+const getDefaultEditorConfig = ({
```

Comment:
I liked that presets were separate files, but unfortunately, the logic with presets grew too complex. In addition to what is in this PR, there is also logic for premium plugins, that made it even more complex. I removed it from PR as it has an unrelated issue, but we'll add it in a separate task.